waterlog/src/Pages/Alert.js

```diff
@@ -1,19 +1,27 @@
 import React, { Component } from 'react';
-//import { BrouserRouter as Router, Route, Link } from "react-router-dom"; 
-import {Provider} from 'react-redux';
+
+import AlertTable from '../Components/AlertTable_';
+import { Provider } from 'react-redux';
 import store from '../store';
 
 class AlertComponent extends Component {
-  render() {
-    return (
-        <Provider store={store}>
-          <div className="App">
-            <header className="App-header"> 
-            </header>
-          </div>
-        </Provider>
-    );
-  }
+	render() {
+		return (
+			<Provider store={store}>
+				<div className="App">
+					<h1 style={{ textAlign: 'center', font: 'Malayalam Sangam MN', color: 'white', padding: '5px' }}>
```

Comment:
General good practice: avoid inline styles and rather use classes defined in your stylesheet

Reply:
inline styles were moved to stylesheet files